pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -842,6 +842,17 @@ type CommonPrometheusFields struct {
 	// +listMapKey=name
 	ScrapeClasses []ScrapeClass `json:"scrapeClasses,omitempty"`
 
+	// Defines the sharding configuration for scraped targets.
+	// It requires enabling the `PrometheusTopologySharding` feature gate.
+	//
+	// WARNING: It's incompatible with the DaemonSet mode for PrometheusAgent.
+	//
+	// This is an *experimental feature*, it may change in any upcoming release
+	// in a breaking way.
+	//
```

Comment:
since the fields have no effect for now, it would be useful to list the implemented features listed in the action plan.

```suggestion
	//
	// Implemented features:
	// [ ] Node selector update when `mode: Topology`.
```	// [ ] External label name when `mode: Topology`.
```	// [ ] Target sharding when `mode: Topology`.
```

Reply:
Thanks @simonpasquier 
Got it.
Added